cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/PhysicalStructureChange.java

```diff
@@ -56,7 +56,6 @@ public abstract class PhysicalStructureChange<T extends Setting> extends CwmsDTO
     private final Double newTotalDischargeOverride;
     private final Double oldTotalDischargeOverride;
     private final String dischargeUnits;
-    private final Double poolElevation;
```

Comment:
I believe you could just keep the poolElevation in this base class. Then in the class that requires it (GateChange), you can override the validator

`@Override
    protected void validateInternal(CwmsDTOValidator validator)
    {
        validator.required(getPoolElevation(), "poolElevation");
        super.validateInternal(validator);
    }`

Reply:
Validation isn't really the reason I did any of this.  The public API for the two change classes is what I was considering when making this change.  poolElevation is required for gate changes, but not for turbines, so for turbines the method `public Double getPoolElevation()` makes sense.  Whereas this doesn't make sense for the gate changes.  We can't override the `getPoolElevation()` to return a double instead, which is why the API is split apart, and since poolElevation is a private field it's also separated with the public API.
If you can think of a better approach to keeping that API that way, let me know and I'd be glad to consider that.

Reply:
I updated it to use Double's, as we had discussed, and had to adjust the GateSetting to do the same.  I like the idea of the validation handling this.